gce/api-servers.tf

```diff
@@ -27,13 +27,37 @@ resource "google_compute_http_health_check" "api" {
   healthy_threshold = "${var.health_check_healthy}"
   unhealthy_threshold = "${var.health_check_unhealthy}"
 }
+resource "google_compute_http_health_check" "api_https" {
+  name = "${var.env}-tsuru-api-https"
+  port = 443
```

Comment:
Does this work? I'd assumed that pointing an HTTP health check at an HTTPS endpoint would fail, because it would try to talk HTTP to something that wanted to speak TLS.